src/playback-watcher.js

```diff
@@ -502,7 +503,7 @@ export default class PlaybackWatcher {
     if (seekable.length &&
         // can't fall before 0 and 0 seekable start identifies VOD stream
         seekable.start(0) > 0 &&
-        currentTime < seekable.start(0) - Ranges.SAFE_TIME_DELTA) {
+        currentTime < seekable.start(0) - (this.liveRangeSafeTimeDelta || Ranges.SAFE_TIME_DELTA)) {
```

Comment:
What about using `this.liveRangeSafeTimeDelta` here always and then in https://github.com/videojs/http-streaming/blob/c9b195b8d8451118960aba2977c1bbc95fda0120/src/videojs-http-streaming.js#L639-L643 defaulting it to `Ranges.SAFE_TIME_DELTA` if a value wasn't provided. Probably by adding an object before `this.options` there.
```js
videojs.mergeOptions({
  liveRangeSafeTimeDelta: Ranges.SAFE_TIME_DELTA
},
this.options_,
{
  seekable: () => this.seekable(),
  media: () => this.masterPlaylistController_.media(),
  masterPlaylistController: this.masterPlaylistController_
});
```

Reply:
Done 👍🏻

Reply:
@gkatsev Are the new changes ok?

Reply:
yes, I think so. We'll re-review and take it for a quick spin but likely good to go.